### PR TITLE
JDK-8207015: jlink javafx.graphics on Windows fails: PluginException: Duplicate resources

### PR DESCRIPTION
**Pull request by kevinrushforth**
[JDK-8207015](https://bugs.openjdk.java.net/browse/JDK-8207015) : jlink javafx.graphics on Windows fails: PluginException: Duplicate resources

### CODE REVIEW

**Comment by kevinrushforth** [comment]
This is now ready for review.

File: build.gradle [hunk 1]
```diff
@@ -5589,6 +5589,12 @@ compileTargets { t ->
         def standaloneSdkDir = "${rootProject.buildDir}/${standaloneSdkDirName}"
         def standaloneLegalDir = "${standaloneSdkDir}/legal"
 
+        def excludeDLLs = []
```
Comment (johanvos) [inline build.gradle]: maybe this should be renamed to excludeDynamicLibs or so, as I think (fear) this issue might surface for other platforms in the future as well.

**Comment by johanvos** [comment]
This doesn't break anything on Linux, will test Mac later.
I have a bigger worry about duplicate resources, but I will post that in the original issue at JBS as I think it relates more to jlink.

**Comment by johanvos** [comment]
works on mac too, so +1

File: build.gradle (hunk 1 above)
Comment (kevinrushforth) [reply build.gradle]: OK, I change it to excludeNativeLibs. The code that populates the list is still specific to Microsoft DLLs, but could be generalized later if there is a need.

**Comment by muralibilla** [comment]
Tested in Windows..Changes looks fine..+1

**Pull request merged**